examples/cli/elan/train.py

```diff
@@ -19,7 +19,7 @@
     ds = elpis.new_dataset('dsy')
     ds.add_directory('/recordings/transcribed', extensions=['eaf', 'wav'])
     ds.auto_select_importer() # Selects Elan because of eaf file.
-    ds.importer.set_setting('tier_name', 'Phrase')
+    ds.importer.set_setting('tier_name', 'tx')
```

Comment:
This change is not necessary. Our canonical tests use the Abui toy corpora which uses 'Phrase as the tier name'